templates/glue-job-S3ToJsonS3.yaml

```diff
@@ -0,0 +1,85 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Out of curiousity: Is this the same template as whats in BridgeDownstream?

Reply:
Yes.